lib/metrics/statsd.js

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var StatsD = require('node-statsd')
+
+var STATSD_PREFIX = 'fxa.auth.'
+
+function StatsDCollector(log) {
+  if (! log) {
+    throw new Error('Log is required')
+  }
+
+  var config = require('../../config')
```

Comment:
hmm the change to support this is massive but possible, config will need to be passed to all these: 
![image](https://cloud.githubusercontent.com/assets/128755/9482649/298242bc-4b64-11e5-84f7-b9b99b391ab2.png)